### ISSUE TITLE
Suggestion - For making Working kitchen For Kakashi Sushi restaurant in legacy roleplay india server  

### ISSUE THREAD

**Issue by Kabir7700** [InZidiuZ/op-framework-issue-tracker#2604]
### Assurance

I have read the `README.md`

### Summary

• I would like to request you for Working kitchen and working restaurant in Restaurant named ‘ KAKSHI SUSHI RESTAURANT’ in legacy roleplay india ! 

•It would be great if we can see working Kitchen and working Japanese restaurant named ‘ KAKASHI SUSHI ‘ in legacy RP india

• I’ve a menu suggestion as well for the restaurant: 

RESTAURANT MENU FOR KAKASHI SUSHI 🍣:- 

—->FOOD ITEMS:- 

•Nigiri sushi
•Sashimi Sushi
•Ramen
•Omu-Rice
•onigiri
•Shrimp tempura
•Sashimi
•Udon noodles

—-> DRINKS :- 

•Soymilk drink
•Green tea
•Sake
•Amazake
•Ramune

if you can add this items to Kakashi Sushi restaurant! It would be great ! 

And please can you start this Kitchen and restaurant as soon as possible ? 

thank you ! 

### Reason

For better RP experience for those who want to continue their RP in Food business ! 
And also for RP progression of civilians and restaurants owner and employees

**Comment by Angadsharma114** [comment]
Yes it’s gonna enhance RP experience and also gonna give different RP to us ! 
And also I love hanging around this place personally ! 
So hopefully we can see this restaurant working as soon as possible

**Comment by Sourabh17-4** [comment]
Just a suggestion as mentioned , if you can add stress releasing  food it’ll be great ! No
And also it’s different restaurant and I’ve seen this in city ! So it’s gonna be good to do RP over there once it’s starts working ! 
And also Good suggestion for RP enhancement !

**Comment by PUSHPA6969** [comment]
a suggestion as mentioned , if you can add stress releasing food it’ll be great ! No
And also it’s different restaurant and I’ve seen this in city ! So it’s gonna be good to do RP 
it’s gonna enhance RP experience and also gonna give different RP to us
and 1 more suggestion make meal cheaper that people will come more and in city only 2 restaurant is available and i been seen that the siting area is so good in KAKASHI so people will spend more time there its gonna give different RP to us
So hopefully we can see this restaurant working as soon as possible

**Comment by Liltears33** [comment]
My name is Tommy Turner the owner of Kissaki Sushi for the international server. I think Mr. Malik is on the right track 100%. These improvements would highly benefit the quality of life within the restaurant. Having a map indicator would also highly benefit the business. Having quality serve can be achieve much easier with the keys and imported food. Would absolutely believe the business would thrive with these changes. Thank you for your time!


- Tommy Turner ~ Kissaki
Liltears#7605

**Comment by FluffyNick** [comment]
Yeh , I absolutely agree with the suggestion. 
Honestly bored of the food in the city weather it’s from Burgershot and bean machine 
Change is necessary and this change seems good ! 
Looking forward  to it

**Comment by seamarob** [comment]
Having food with perks will definitely help bring back the regular use of restaurants, fine dining or fast food. This addition while bringing more traffic to the business will give more motivation for employees of those businesses as well.

**Comment by coalaura** [comment]
closing in favor of this #3423
